File: Java/Easy/Accepted/cells-with-odd-values-in-a-matrix_7.java
class Solution {
    public int oddCells(int m, int n, int[][] indices) {
        
        int[] row = new int[m];
        int[] col = new int[n];
        for (int[] idx : indices) {
            row[idx[0]]++;
            col[idx[1]]++;
        }
        int odd = 0;
        for (int r = 0; r < m; r++) {
            for (int c = 0; c < n; c++) {
                if ((row[r] + col[c]) % 2 == 1) {
                    odd++;
                }
            }
        }
        return odd;
    }
}

File: Rust/Easy/Accepted/cells-with-odd-values-in-a-matrix_7.rs
impl Solution {
    pub fn odd_cells(m: i32, n: i32, indices: Vec<Vec<i32>>) -> i32 {
        
        let mut matrix = vec![vec![0; n as usize]; m as usize];
        for index in indices {
            let (i, j) = (index[0] as usize, index[1] as usize);
            for k in 0..n as usize {
                matrix[i][k] += 1;
            }
            for k in 0..m as usize {
                matrix[k][j] += 1;
            }
        }
        let mut count = 0;
        for row in matrix {
            for cell in row {
                if cell % 2 == 1 {
                    count += 1;
                }
            }
        }
        count
    }
}